### DISCUSSION TITLE
Application under review

### DISCUSSION THREAD

**Discussion by Himanshunitrr** [github/india#635] [Students]
Its almost a week now and its still showing that my application is under review. Did i fill in something wrong? Any help/ suggestion will be highly appreciated.

**Comment by shobhi1310** [comment]
I too had the same query, I am a bit relieved to know that at least there is someone else also who feels the same.

**Comment by DhruvMehta-code** [comment]
I also have same query  , When will we know about our application status change from under review ?

**Reply by Himanshunitrr** [reply]
Don't know.

**Comment by punitjain03** [comment]
We are adding more organisation for you all to participate and therefore the status is under review.

**Reply by DhruvMehta-code** [reply]
Thank you !!

**Comment by bala-ceg** [comment]
I also have the same query, When will we know about our application status change from under review ?
Is there any way to speed up our application status?
